python/consumer/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM registry.access.redhat.com/ubi8/python-39:latest
+
+USER root
+
+# Add strimzi user with UID 1002
+# The user is in the group 0 to have access to the mounted volumes and storage
+RUN useradd -r -m -u 1002 -g 0 strimzi
+
+ARG version=latest
+ENV VERSION ${version}
+
+ADD kafka_consumer.py $HOME
+ADD kafka_consumer_config.py $HOME
+ADD requirements.txt $HOME
+
+RUN pip3 install -r requirements.txt
+
+USER 1002
+
+CMD ["python3", "/opt/app-root/src/kafka_consumer.py"]
```

```suggestion
CMD ["python3", "${HOME}kafka_consumer.py"]
```
Would something like this work?

---

Results in the following error. It seems to reed the home directory and the variable...
`python3: can't open file '/opt/app-root/src/${HOME}kafka_consumer.py': [Errno 2] No such file or directory`

---

`CMD ["python3", "kafka_consumer.py"]` works